client/src/views/util.js

```diff
@@ -8,13 +8,17 @@ const DEFAULT_PRECISION = 0
 
 const pad = n => n < 10 ? '0'+n : n
 
-export const formatTime = unix => {
+export const formatTimezone = unix => {
+  const tzOffset = new Date(unix*1000).getTimezoneOffset() * -1;
+  return `GMT${tzOffset == 0 ? '' : (tzOffset < 0 ? '' : '+') + (tzOffset/60)}`
+}
+
+export const formatTime = (unix, with_tz) => {
```

Comment:
`with_tz` should default to true, so that the other places that display timestamps include the timezone.

Reply:
The default still needs to change here.